snapcraft/internal/pluginhandler/__init__.py

```diff
@@ -470,6 +469,16 @@ def prime(self, force=False):
                 _migrate_files(system, system_dependency_paths, '/',
                                self.primedir, follow_symlinks=True)
 
+        # If we are classic we need to patch the elf files for which
+        # prime will be called with instructions to do so
```

Comment:
I don't quite understand this sentence. "elf files for which prime will be called" ? We're _in_ prime.

Reply:
meaning, this `prime` was called with instructions to patch; no matters anymore as I'll be moving the parameter to the class/object.